xml/security-troubleshooting_tls.xml

```diff
@@ -156,9 +156,9 @@ ansible-playbook -i hosts/verb_hosts tls-percona-reconfigure.yml</screen>
    </step>
    <step>
     <para>
-     Verify Percona cluster status
+     Verify Percona cluster status on a controller node
     </para>
-    <screen>&prompt.ardana;sudo mysql -e show status</screen>
+    <screen>&prompt.ardana;sudo mysql -e 'show status'</screen>
```

Comment:
Do you type 'show status' in quotes in this command?

Reply:
yes, exactly